eventlet/patcher.py

```diff
@@ -355,16 +467,20 @@ def after_fork():
     # threading.get_ident(). Force the Python implementation of RLock which
     # calls threading.get_ident() and so is compatible with eventlet.
     import threading
+
     threading.RLock = threading._PyRLock
 
     # Issue #508: Since Python 3.7 queue.SimpleQueue is implemented in C,
     # causing a deadlock.  Replace the C implementation with the Python one.
     import queue
+
     queue.SimpleQueue = queue._PySimpleQueue
 
     # Green existing locks _after_ patching modules, since patching modules
     # might involve imports that create new locks:
-    _green_existing_locks(original_rlock_type)
+    for name, _ in modules_to_patch:
```

Comment:
And this is a meaningful change, and everything else is syntax tweaks, sorry again.